scvi/inference/trainer.py

```diff
@@ -220,6 +222,34 @@ def create_posterior(self, model=None, gene_dataset=None, shuffle=False, indices
         return type_class(model, gene_dataset, shuffle=shuffle, indices=indices, use_cuda=self.use_cuda,
                           data_loader_kwargs=self.data_loader_kwargs)
 
+    def get_dataset_information(self, save_imputed=False, file_name_imputation='imputed_values',
+                                save_shape_genes_by_cells=False, save_latent=False, file_name_latent='latent_space'):
```

Comment:
The name `get_dataset_information` is really vague. Better to give it a name that describes what this function actual does. Also, good to give a docstring.

Reply:
Wouldn't get_all_latent_and_imputed_values be a little bit too long?

Reply:
It's an improvement. Short and clear is the best, but too long is better than too vague.